monitors/tests/test_icat_monitor.py

```diff
@@ -0,0 +1,82 @@
+"""
+Unit tests for the ICAT monitor
+"""
+
+import unittest
+import datetime
+from mock import Mock
+
+import monitors.icat_monitor as icat_monitor
+from monitors.settings import INSTRUMENTS
+
+
+# pylint:disable=too-few-public-methods
+class DataFile(object):
```

Comment:
Is this the same form that is returned by icat?

Reply:
Similar form. Python ICAT will return an object with the fields as its attributes. Obviously there will be more fields.

Reply:
Okay cool this is good to know 👍 If we need to do more things like this in the future we should move this to be a test helper somewhere near the `ICATClient` but it's fine here for now as we don't have a reuse case just yet